_posts/2020-11-25-#19858.md

```diff
@@ -10,13 +10,47 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
 
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
+- An [eclipse attack](https://bitcoinops.org/en/topics/eclipse-attacks/) is
+  when an adversary is able to isolate a victim from the bitcoin network by
+  overtaking all of its connections.
+
+- Block-relay-only connections are a type of connection where two nodes do not
+  participate in transaction or address relay, and solely relay blocks. By
+  obfuscating network topology, they help mitigate eclipse attacks.
+
+- Block-relay-only connections were introduced in
+  [#15759](https://github.com/bitcoin/bitcoin/pull/15759). After these changes,
+  nodes by default open two outbound block-relay-only connections on startup.
+  We discussed this PR in a [previous review
+  club](https://bitcoincore.reviews/15759.html) if you are interested in
+  learning more.
+
+- This weeks PR, [#19858](https://github.com/bitcoin/bitcoin/pull/19858)
```

Comment:
s/weeks/week's/ and add a comma after 19858 (or omit the first comma)

Reply:
done